AirCasting/CoreData/MeasurementStreamEntity.swift

```diff
@@ -49,6 +49,9 @@ public class MeasurementStreamEntity: NSManagedObject, Identifiable {
     public var localID: MeasurementStreamLocalID {
         MeasurementStreamLocalID(id: objectID)
     }
+    public var thresholds: [Int32] {
+     [thresholdVeryLow, thresholdLow, thresholdMedium, thresholdHigh, thresholdVeryHigh]
```

Comment:
Perhaps this could be the thresholds struct and not an array? Using array in this case seems prone to error and we already know there can be only this limited set of 5 thresholds, so using a struct (or enum or dictionary) seems like a good fit. If that changes in the future, for example we will have 4 thresholds, it will be easier to edit without errors caused by `thresholds[4]`.